### PR TITLE
add chinese translation to EnhanceAPI

### PR DESCRIPTION
**Pull request by zilongshanren**

### CODE REVIEW

**Comment by zilongshanren** [comment]
@pandamicro  please review this PR. thanks.

File: cocos/platform/android/CCEnhanceAPI-android.h [hunk 1]
```diff
@@ -24,75 +24,117 @@
...
+     * 此API可以通过设置节能模式来降低设备的功耗。
+     * 
+     *@return  @~english -1: fail to get temperature
      * - 0: normal temperature
      * - 1: high
      * - 2,3: very high
+     * @~chinese -1:没有得到温度
+     * - 0:正常温度
+     * - 1:温度过高
+     * 3 - 2:温度非常高
```
Comment (pandamicro) [inline cocos/platform/android/CCEnhanceAPI-android.h]: `- 2,3`

File: cocos/platform/android/CCEnhanceAPI-android.h [hunk 2]
```diff
@@ -24,75 +24,117 @@
...
     /**
-     *@brief The function is used to set power saving mode for true or false.
+     *@brief @~english The function is used to set power saving mode for true or false.
      * The purpose of the API is reducing Power consumption when set power saving mode for true.
      *
-     *@return  -1: fail to get temperature
+     * @~chinese 该函数用于设置是否开启省电模式。
+     * 此API可以通过设置节能模式来降低设备的功耗。
+     * 
+     *@return  @~english -1: fail to get temperature
```
Comment (pandamicro) [inline cocos/platform/android/CCEnhanceAPI-android.h:132]: `- -1`

File: cocos/platform/android/CCEnhanceAPI-android.h [hunk 3]
```diff
@@ -24,75 +24,117 @@
...
      * - 0: normal temperature
      * - 1: high
      * - 2,3: very high
+     * @~chinese -1:没有得到温度
```
Comment (pandamicro) [inline cocos/platform/android/CCEnhanceAPI-android.h]: `- -1`

File: cocos/platform/android/CCEnhanceAPI-android.h [hunk 4]
```diff
@@ -24,75 +24,117 @@
...
+     *@brief   @~english The function is used to get temperature level(0-3).
      * The purpose of the API is get temperature status of the device.
      * Temperature 0 means normal temperature and 1-3 means different temperature level.
      * 1 is the minimum high temperature level and 3 is the maximum high temperature level.
-     *@return  -1: fail to get temperature.
+     * @~chinese 此函数是用于获取设备的温度水平(0 - 3)。
+     * API的目的是获得设备的温度状态。
+     * 温度0意味着正常温度和1 - 3处于不同的温度水平。
+     * 1 is the minimum high temperature level and 3 is the maximum high temperature level.
+     *@return  @~english -1: fail to get temperature.
```
Comment (pandamicro) [inline cocos/platform/android/CCEnhanceAPI-android.h]: `- -1`

File: cocos/platform/android/CCEnhanceAPI-android.h [hunk 5]
```diff
@@ -24,75 +24,117 @@
...
      * - 0: normal temperature
      * - 1: high
      * - 2,3: very high
+     * @~chinese -1: 无法获取温度。
```
Comment (pandamicro) [inline cocos/platform/android/CCEnhanceAPI-android.h]: `- -1`

File: cocos/platform/android/CCEnhanceAPI-android.h (hunk 2 above)
Comment (zilongshanren) [inline cocos/platform/android/CCEnhanceAPI-android.h:132]: thanks, already fixed.

**Pull request merged**